### DISCUSSION TITLE
Projects Beta: receive (web/email) notifications for project updates

### DISCUSSION THREAD

**Discussion by steebchen** [community/community#10077] [Projects and Issues]
Jira sends an email when a ticket status is edited, for example, from "To Do" to "In Progress". This is really useful for PMs, but not for most people probably, so it would be great if you could opt-in to receiving notifications (ideally you could select between web/email/both) for whenever issues are updated.

**Comment by lisaattrillium** [comment]
This would be highly valuable for cross-functional task management. and yes please to unsubscribing to notifications!
